src/pages/index.js

```diff
@@ -66,14 +118,48 @@ addButton.addEventListener("click", () => {
   cardsModal.open();
 });
 
+editAvatarButton.addEventListener("click", () => {
+  avarEditModal.open();
+});
+
 const handleImageClick = (name, link) => {
   imageModal.open({ name: name, link: link });
 };
+const handlerRemoveCard = (card, id) => {
+  preDeletepoup.open(card, id);
+};
 
 const createCard = (data) =>
-  new Card(data, "card-template", handleImageClick).getCardElement();
+  new Card(
+    data,
+    "card-template",
+    handleImageClick,
+    handlerRemoveCard,
+    api
+  ).getCardElement();
 
 profileModal.setEveentListeners();
 cardsModal.setEveentListeners();
 imageModal.setEveentListeners();
-gallerySection.renderItems();
+avarEditModal.setEveentListeners();
+preDeletepoup.setEveentListeners();
+
+api
+  .getInitialCards()
+  .then((initialCards) => {
+    gallerySection.setItems(initialCards);
+    gallerySection.renderItems();
+  })
+  .catch((err) => {
+    console.log(err);
+  });
+
+api
+  .getUserData()
+  .then((response) => {
+    userInfo.setUserInfo(response.name, response.about);
+    userInfo.setAvatarLink(response.avatar);
+  })
+  .catch((err) => {
+    console.log(err);
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors